src/libstd/panicking.rs

```diff
@@ -38,6 +39,7 @@ enum Handler {
 
 static HANDLER_LOCK: StaticRwLock = StaticRwLock::new();
 static mut HANDLER: Handler = Handler::Default;
+static mut FIRST_PANIC: AtomicBool = AtomicBool::new(true);
```

Comment:
This can actually just be a plain old `static` rather than a `static mut` (and all access will be safe!)